portfolio/src/main/webapp/index.html

```diff
@@ -8,8 +8,9 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1> Toya's Portfolio </h1>
```

Comment:
There's a couple extra white spaces, but otherwise it's all good!
```suggestion
      <h1>Toya's Portfolio</h1>
```